build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/dotnet:2.0.0-sdk as builder
+FROM microsoft/dotnet:2.1-sdk as builder
```

Comment:
LGTM, this is because we cross compile for both `netcoreapp2.0` and `netcoreapp2.1` now 👍

Reply:
Yeah, saw that while building. Great to do in the Dockerfile as well!